templates/schedule.html

```diff
@@ -28,6 +30,7 @@
             <td colspan={{ cell.width }} rowspan={{ cell.height }}>
               {% if cell.value.url %}
               <a href="{{ cell.value.url }}">{{ cell.value.text }}</a>
+              <small class="speaker">{{ cell.value.speaker }}</small>
```

Comment:
I think it might be good to add a `<br/>` to put the speaker name on a new line – as it stands, the columns end up changing widths when I change the toggle. Adding a `<br/>` keeps the columns the same width, which makes for a less jarring transition.